stock_by_warehouse/static/src/scss/main.scss

```diff
@@ -1,3 +1,9 @@
 .js_product_warehouse, .js_available_info{
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+@import url('enterprise/web_enterprise/static/src/scss/fields.scss');
```

Comment:
Are you sure you need to rely on enterprise? Why? This module doesn't depend on any enterprise module